### PR TITLE
Create pre-commit hook to automate prettier integration.

### PR DESCRIPTION
**Pull request by rodrigooler**
Issue: https://github.com/RebirthDB/rebirthdb-desktop/issues/11

- [x] add jest --env=node;
- [x] add lint-staged;
- [x] add husky;
- [x] fix standard-html;
- [x] fix index.html;
- [x] fix tests;
- [x] run lint;
- [x] improve lint-staged;
- [x] test precommit;

### CODE REVIEW

**Comment by rodrigooler** [comment]
@sagivf I would like suggestions about PR

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "package-all": "npm run build && build -mwl",
     "package-linux": "npm run build && build --linux",
     "package-win": "npm run build && build --win --x64",
-    "standard": "standard --fix 'app/**/*.js' && yarn run lint",
+    "precommit": "lint-staged",
```
Comment (rodrigooler) [inline package.json]: @sagivf to run the prettier or the standard first?

File: package.json [hunk 2]
```diff
@@ -20,7 +21,7 @@
     "test-all": "npm run lint && npm run flow && npm run build && npm run test && npm run test-e2e",
     "test-e2e": "cross-env NODE_ENV=test BABEL_DISABLE_CACHE=1 node --trace-warnings -r babel-register ./internals/scripts/RunTests.js e2e",
     "test-watch": "npm test -- --watch",
-    "lint": "prettier 'app/**/*.js' --write"
+    "lint": "prettier 'app/**/*.js' --write && yarn standard && yarn standard-html && yarn standard-markdown"
```
Comment (rodrigooler) [inline package.json]: @sagivf  what do you think?

**Comment by grandquista** [comment]
It looks like this is putting tabs in html with spaces elsewhere. I’m not sure what the overall team expects just noting it. Otherwise good PR.

**Comment by rodrigooler** [comment]
It was because of the script `standard-html`, I'm waiting for other members to comment to see if it solves the issue issue.

File: package.json (hunk 1 above)
Comment (sagivf) [reply package.json]: @rodrigooler sounds right to me

File: package.json (hunk 2 above)
Comment (sagivf) [reply package.json]: Looks good

File: package.json [hunk 3]
```diff
@@ -112,5 +115,11 @@
     "env": [
       "jest"
     ]
+  },
+  "lint-staged": {
+    "*.{html,md,js}": [
+      "yarn lint",
+      "git add"
```
Comment (sagivf) [inline package.json]: never user lint-stages but I think I get the idea. what us the `git add` for?

**Pull request merged**